Rewording of optionnal challenges (03-sql-joins) #271 (checkout) (#276)
datacarpentry/sql-ecology-lesson: _episodes/03-sql-joins.md

> 
> 1. How many plots from each type are there?  
> 

> > 2. How many specimens are of each sex are there for each year? How many in each year could not be sexed? 

Reviewer: ```suggestion
> 2. How many specimens are of each sex are there for each year, including those whose sex is unknown?
```
